--- src-original/setup.py       2021-09-17 21:15:10.000000000 +0000
+++ src/setup.py        2023-04-03 11:54:47.113267448 +0000
@@ -11,1 +11,1 @@
-from setuptools import Extension, setup
+from setuptools import Extension, setup , find_packages
@@ -30,1 +30,11 @@
-# Macros to enable Cython coverage
+# Macros to extract he current verion
+def extract_version(versionfile):
+    version = None
+    with open (versionfile) as fi:
+	    for line in fi:
+	        if line.startswith("__version__"):
+	           _, version = line.split("version =")
+	           version = version.strip()[1:-1]  # Remove quotation characters.
+	           break
+    return version
+# Macros to enable Cython coverage
@@ -55,1 +55,16 @@
-    ext_modules=extensions,
+    name='cartopy',
+    version=extract_version("lib/cartopy/_version.py"), #Call Extract version from _version.py file,
+    packages=find_packages(
+         where='lib',
+         include=['*'],  
+     ),
+    package_dir={"": "lib"},
+    ext_modules=extensions,
+    py_modules=['crs'],
+    install_requires=[
+        "pyproj",
+        "matplotlib",
+        "shapely",
+	 "packaging",
+	 "pyshp",
+    ],

